contracts/feature-tests/scenario-tester/tests/st_blackbox_test.rs

```diff
@@ -1,36 +1,65 @@
-use multiversx_sc_scenario::imports::*;
+use multiversx_sc_scenario::{imports::*, scenario_model::U64Value};
 use num_bigint::BigUint;
 
 use scenario_tester::*;
 
-const ADDER_PATH_EXPR: &str = "mxsc:output/scenario-tester.mxsc.json";
+const SC_SCENARIO_TESTER_PATH_EXPR: &str = "mxsc:output/scenario-tester.mxsc.json";
 
 const OWNER: AddressExpr = AddressExpr("owner");
-const SC_ADDER: ScExpr = ScExpr("scenario-tester");
+const OTHER: AddressExpr = AddressExpr("other");
+const SC_SCENARIO_TESTER: ScExpr = ScExpr("scenario-tester");
 const CODE_EXPR: MxscExpr = MxscExpr("output/scenario-tester.mxsc.json");
 
 fn world() -> ScenarioWorld {
     let mut blockchain = ScenarioWorld::new();
     blockchain.set_current_dir_from_workspace("contracts/examples/scenario-tester");
 
-    blockchain.register_contract(ADDER_PATH_EXPR, scenario_tester::ContractBuilder);
+    blockchain.register_contract(
+        SC_SCENARIO_TESTER_PATH_EXPR,
+        scenario_tester::ContractBuilder,
+    );
     blockchain
 }
 
 #[test]
 fn st_blackbox() {
     let mut world = world();
     let owner_address = "address:owner";
+    let other_address = "address:other";
+
     let st_contract = ContractInfo::<scenario_tester::Proxy<StaticApi>>::new("sc:scenario-tester");
 
     world.start_trace();
 
-    world.set_state_step(
-        SetStateStep::new()
-            .put_account(owner_address, Account::new().nonce(1))
-            .new_address(owner_address, 1, "sc:scenario-tester"),
+    world
```

Comment:
Let's have a test with just these 2: a set stater followed by check state. This will help us in the long run.

We can test with and without .commit() at the end.